### PR TITLE
Setup mobile first portfolio

### PR DESCRIPTION
**Pull request by kene-creator**
# Mobile view portfolio

### Steps covered

- Created landing page header
- Add added linters

### CODE REVIEW

**Comment by Ismailco** [comment]
@kene-creator, you have some linter errors make sure you fix them ASAP

**Review by Ismailco — COMMENTED** [review]
Hi @kene-creator, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README copy.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
```
Comment (frank1738) [inline README copy.md:15]: - [ ] Because you have not deployed the project yet, You can remove the live demo link

File: README copy.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+-Webhint
+-StyleLint
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@kene-creator](https://github.com/kene-creator)
+- Twitter: [@kay_yung1](https://twitter.com/kay_yung1)
+- LinkedIn: [Kenechuwku Agagwu](https://linkedin.com/in/kenechukwuagagwu)
```
Comment (frank1738) [inline README copy.md:34]: - [ ] Your LinkedIn link is not working, would you please remove it if it is not available

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @kene-creator 😃 , Im [frank](@frank1738) your Code Reviewer

Good job correcting linter errors as requested by the previous reviewer
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Have used the right GitHub flow ✅
- Linter checks passed ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @kene-creator , 

Good job so far! 👏👏👏
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No Linters Errors ✔️
- Good commit messages ✔️
- PR has a good title and summary ✔️

 ## Required Changes ♻️
 
- [ ]  _**_Check The Comments Below_**_

## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -0,0 +1,117 @@
+:root {
+  --header-font: "Poppins";
+}
```
[the remark this answers is not in the document]
Comment (kene-creator) [reply style.css:3]: I was still working on the section. Currently I'm working on sections and I'll implement that

File: index.html [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z"
+            fill="#6070FF"
+          />
+        </svg>
+      </nav>
```
[the remark this answers is not in the document]
Comment (kene-creator) [reply index.html:29]: They are no extra sections in the header

**Review by omarsalem7 — APPROVED** [review]
Hi @kene-creator 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**